contracts/smart-contract-wallet/aa-4337/core/BasePaymaster.sol

```diff
@@ -22,6 +22,9 @@ abstract contract BasePaymaster is IPaymaster, Ownable {
     }
 
     function setEntryPoint(IEntryPoint _entryPoint) public onlyOwner {
+        // review - 0 address check required?
+        require(address(_entryPoint) != address(0), "entryPoint cannot be 0");
```

Comment:
0 address check required

Reply:
this will be obsolete so it doesn't matter